### PR TITLE
Develop gradio app

### PR DESCRIPTION
**Pull request by JSchmie**
Added an entirely new backend, using a config.yml to handle the setup of the server:
Added multiprosseccing Option to unload and reload the AI Models in the Background so if there are not used they do not block RAM space.

### CODE REVIEW

File: scraibe/app/app.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    
+"""
+
+
+####
+# Gradio Interface
+####
+
+from threading import Thread
+
```
Comment (Tryndaron) [inline scraibe/app/app.py:24]: Add Docstring which states the following imports are from internal Modules

Comment (JSchmie) [reply scraibe/app/app.py:38]: Done

**Comment by JSchmie** [comment]
Status?

**Comment by JSchmie** [comment]
The Gradio Web App will be moved to another Repository, which can be found here: https://github.com/JSchmie/ScrAIbe-WebUI

**Pull request merged**